### ISSUE TITLE
No cop checking for space after (or within) block arguments?

### ISSUE THREAD

**Issue by bquorning** [bbatsov/rubocop#1627]
```rb
[].each { |x|puts x }
```
I cannot find an existing cop that would complain about the missing space after `|x|`.

I am thinking that perhaps this should be a responsibility of the `SpaceInsideBlockBraces` cop, or – since that cop is maybe growing too large – if there should be a new `SpaceAroundBlockArguments` cop that checks the spacing on both sides of the `|x|`.

**Comment by bquorning** [comment]
Maybe the spacing inside the arguments “block” could be checked by the same cop. In the following example, there should be only one space after the comma, and no space before `x` and after `y`:

```rb
{}.each { | x,  y |puts x }
```

**Comment by bbatsov** [comment]
I agree it'd be great if we had both checks. Guess @jonas054 can look into implementing them.
